src/QuadVelocityController.cpp

```diff
@@ -292,8 +310,44 @@ bool QuadVelocityController::update(const ros::Time& time,
             4,
             col_height)
             / voltage;
-    uav_command.data.pitch = pitch_output;
-    uav_command.data.roll = roll_output;
+
+    if(xy_mixer_ == "4dof") {
+        uav_command.data.pitch = pitch_output;
+        uav_command.data.roll = -roll_output;
+    }
+    else if(xy_mixer_ == "6dof") {
+        double x_accel = pitch_output;// + setpoint_.motion_point.accel.linear.x;
```

Comment:
Why is this commented out?

Reply:
It's not safe to use feedforward because the velocity estimates are not reliable enough.